stacks-cli/data-scaffold-example.yml

```diff
@@ -0,0 +1,45 @@
+input:
+  log:
+    level: info
+    format: text
+    colour: true
+  directory:
+    working: stacks # Update: Target directory for the scaffolded project
+    export: "~"  # Update: Path to your Stacks CLI installation
+  business:
+    company: mycompany # Update: Used for resource naming
+    domain: mydomain  # Update: Used for environment & Terraform state key naming
+    component: none
+  cloud:
+    platform: azure
+  network:
+    base:
+      domain:
+        internal: none
+        external: none
```

Comment:
Rather than none, may be clearer if we just don't have these options in our template? CLI will still generate without them
```suggestion
```